1.1/percentile.go

```diff
@@ -16,13 +18,26 @@ func createData(rand *rand.Rand, n uint) []uint {
 }
 
 type percentile func([]uint) uint
+type seedData []uint
+
+func (a seedData) Len() int { return len(a) }
+
+func (a seedData) Swap(i, j int) { a[i], a[j] = a[j], a[i] }
+
+func (a seedData) Less(i, j int) bool { return a[i] < a[j] }
```

Comment:
As you've noticed, sorting in go is wierd compared to other languages.  Fufilling the sortable interface is excellent practice, but probably overkill.  Take a look at:

https://golang.org/pkg/sort/#Slice